packages/ember-metal/lib/computed.js

```diff
@@ -30,36 +30,35 @@ function UNDEFINED() { }
 //
 
 /**
-  A computed property transforms an object's function into a property.
+  A computed property transforms a hash with object's accessor function(s) into a property.
```

Comment:
`hash` is probably confusing for some javascripters. 'object' or 'object literal' is more accurate.